source/adapter.js

```diff
@@ -1069,22 +1069,24 @@ if (navigator.mozGetUserMedia) {
         element.setStreamId(streamId);
       }
       var newElement = document.getElementById(elementId);
-      newElement.onplaying = (element.onplaying) ? element.onplaying : function (arg) {};
-      newElement.onplay    = (element.onplay)    ? element.onplay    : function (arg) {};
-      newElement.onloadedmetadata    = (element.onloadedmetadata)    ? element.onloadedmetadata    : function (arg) {};
-      newElement.onclick   = (element.onclick)   ? element.onclick   : function (arg) {};
-      if (isIE) { // on IE the event needs to be plugged manually
-        newElement.attachEvent('onplaying', newElement.onplaying);
-        newElement.attachEvent('onplay', newElement.onplay);
-        newElement.attachEvent('onloadedmetadata', newElement.onloadedmetadata);
-        newElement._TemOnClick = function (id) {
-          var arg = {
-            srcElement : document.getElementById(id)
-          };
-          newElement.onclick(arg);
-        };
+
+      //Event forwarding
+      for(var property in element) {
+        if(property.slice(0,2) == "on" && element[property] != null) {
+          AdapterJS.addEvent(newElement, property.slice(2), element[property]);
+        }
       }
-      
+      if(!isIE) {
+        var nonEnumerablesToForward = ['onclick'];
+        
```

Comment:
I couldn't find a way to iterate over the non-enumerable properties. I'm using an array containing all the properties one could be interested in forwarding.